tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java

```diff
@@ -1738,8 +1738,8 @@ private String handleDateFormatItem(String xpath, String value) {
         String calendar = parts.findAttributeValue("calendar", "type");
 
         if (parts.contains("dateTimeFormat")) {
-            String dateFormatXPath = cldrFile.getWinningPath(xpath.replaceAll("dateTimeFormat", "dateFormat"));
-            String timeFormatXPath = cldrFile.getWinningPath(xpath.replaceAll("dateTimeFormat", "timeFormat"));
+            String dateFormatXPath = cldrFile.getWinningPath(xpath.replaceAll("dateTimeFormat", "dateFormat").replaceAll("atTime", "standard"));
```

Comment:
We might want to general multiple examples, one with atTime and one with standard. But that can be a followup.

Reply:
This will already generate examples for each of the date-time combining pattern styles (the standard and the atTime); the code above is about getting the date and time formats to use in those examples, since there is no type="atTime" in the date & time patterns. But what we probably *do* want to do is generate multiple examples for the "standard" style, including  one with a single time (as generated by the above), and one with a time range(for intervalFormats). I as thinking about that and filed https://unicode-org.atlassian.net/browse/CLDR-15622 for this.